tests/mpls_input/appl_db.json

```diff
@@ -1,47 +0,0 @@
-{
```

Comment:
this one too.. is it required to removed? Does it affect multi ASIC test cases?

Reply:
no, will bring back